app/cdap/components/GridTable/index.tsx

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Table, TableBody, TableHead, TableRow } from '@material-ui/core';
+import MyDataPrepApi from 'api/dataprep';
+import { directiveRequestBodyCreator } from 'components/DataPrep/helper';
+import DataPrepStore from 'components/DataPrep/store';
+import DataPrepActions from 'components/DataPrep/store/DataPrepActions';
+import If from 'components/shared/If';
+import LoadingSVG from 'components/shared/LoadingSVG';
+import { default as React, useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import { objectQuery } from 'services/helpers';
+import BreadCrumb from './components/Breadcrumb';
+import { GridHeaderCell } from './components/GridHeaderCell';
+import { GridKPICell } from './components/GridKPICell';
+import { GridTextCell } from './components/GridTextCell';
+import Box from '@material-ui/core/Box';
+import { useStyles } from './styles';
+import { flatMap } from 'rxjs/operators';
+import { forkJoin } from 'rxjs/observable/forkJoin';
+
+const GridTable = () => {
+  const { wid } = useParams() as any;
+  const params = useParams() as any;
+  const classes = useStyles();
+
+  const [loading, setLoading] = useState(false);
+  const [headersNamesList, setHeadersNamesList] = React.useState([]);
```

Comment:
don't use React.useState - this should just be useState

Reply:
Done